src/components/manage/Blocks/Listing/View.jsx

```diff
@@ -12,6 +12,7 @@ const View = (props) => {
     <div
       className={cx('block listing', data.variation || 'default', className)}
     >
+      {data.headline && <h2 className="headline">{data.headline}</h2>}
```

Comment:
The heading level is problematic (sometimes h3 is more appropriate, etc, it really depends on the context, and I think it's tough to have semantically correct heading levels in Volto's block system, unless we ask the editor specifically to establish the heading level of that block.

Reply:
Added the heading level in the last commit. What about now?

<img width="374" alt="image" src="https://user-images.githubusercontent.com/486927/188419928-fde93acd-149b-47d6-9639-32139759481a.png">